src/javascripts/todo-list.js

```diff
@@ -138,4 +138,4 @@ class TodoList {
   }
 }
 
-export default TodoList
+return TodoList
```

Comment:
NodeJS uses CommonJS Module syntax (module.exports) not ES6 module syntax (export keyword)